.changeset/tender-pigs-rest.md

```diff
@@ -0,0 +1,26 @@
+---
+"@comet/admin-theme": minor
+---
+
+Add custom `Typography` variants for displaying inline lists
+
+```
+<Typography variant="list">
+    <Typography variant="listItem">Lorem ipsum</Typography>
+    <Typography variant="listItem">Lorem ipsum</Typography>
+    <Typography variant="listItem">Lorem ipsum</Typography>
+</Typography>
+```
+
+Hint: To use the custom variants without getting a type error, you must adjust the `tsconfig.json` in your project:
+
+```diff
+{
+    // ...
+    "compilerOptions": {
+        // ...
++       "typeRoots": ["node_modules/@types", "node_modules/@comet/admin-theme"],
```

Comment:
See https://github.com/vivid-planet/comet/pull/1840/files#r1527506312

But I'd suggest to first merge #1840 to next, then test it and then merge this PR to main once we know it works.

Reply:
Done https://github.com/vivid-planet/comet/pull/1842/commits/440a1c4b504526a44de4104c53b394b4d198183a

Reply:
Please adapt the changeset accordingly

Reply:
Done ac6dcc4cce42844b0f1e7d0ac50512948e43a192